pages/musicalbum/_itemId/index.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <item-cols>
+    <template #left>
+      <v-row justify="center" justify-md="start">
+        <v-col cols="7" md="3">
+          <card :item="item" no-text no-margin />
+        </v-col>
+        <v-col cols="12" md="9">
+          <h1
+            class="text-h4 font-weight-light"
+            :class="{ 'text-center': !$vuetify.breakpoint.mdAndUp }"
+          >
+            {{ item.Name }}
+          </h1>
+          <h2
+            v-if="item.AlbumArtist"
+            class="text-subtitle-1 text-truncate mt-2"
+            :class="{ 'text-center': !$vuetify.breakpoint.mdAndUp }"
+          >
+            {{ $t('byArtist') }}
+            <nuxt-link
+              tag="span"
+              class="link"
+              :to="getItemDetailsLink(item.AlbumArtists[0])"
+            >
+              {{ item.AlbumArtist }}
+            </nuxt-link>
```

Comment:
How could we get the browser to use it as a link (ie: allow right click + copy link or just displaying it when the mouse is over it). I think we need to remove the tag but then it goes back to a default link design. What can be done?

Reply:
Made the link class a global one. nuxt-links that we want to act as span now must have the link class.